pkg/api/universal/conversation.go

```diff
@@ -16,6 +16,8 @@ package universal
 import (
 	"context"
 
+	piiscrubber "github.com/aavaz-ai/pii-scrubber"
```

Comment:
Maybe this is a fine pii scrubber default, but what is the long term plan for users to pick/choose their own scrubber?

Reply:
Long term we should consider making PII scrubbers first class components. There are many good integrations such as [project Presidio](https://github.com/microsoft/presidio), AWS Comprehend and others.